### PR TITLE
Added support for Dimension selection and MS Teams sending.

### PR DESCRIPTION
**Pull request by stevemac007**
You can now deploy the function and override the dimension the cost data is grouped by.

For example, I deploy this as follows.

	serverless deploy --stage="service" \
		--param="group=SERVICE" \
		--param="group_length=20" \
		--param="teams_url=https://abc.webhook.office.com/webhookb2/<hook> \

	serverless deploy --stage="account" \
		--param="group=LINKED_ACCOUNT" \
		--param="group_length=10" \
		--param="teams_url=https://abc.webhook.office.com/webhookb2/<hook>" \

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,42 @@ If you have AWS credits on your account and want to see them taken into account
...
+If you have and AWS Organisation established and would like to see a breakdown by account, you can override the default dimensions with parameters: 
+
+    ```
+    serverless deploy \
+        --param "slack_url=https://hooks.slack.com/services/xxx/yyy/zzzz" \
+        --param "group=LINKED_ACCOUNT" \
+        --param "group_length=15"
+    ```
+
+Possible value for `group` are:
```
Comment (iandees) [inline README.md:73]: Can you include a link to the AWS docs where these groups come from?

Comment (stevemac007) [reply README.md:73]: I actually got them from the error message when I specified an invalid value.

Comment (stevemac007) [reply README.md:73]: From - https://docs.aws.amazon.com/cli/latest/reference/ce/get-cost-and-usage.html#options

> Valid values for the DIMENSION type are AZ , INSTANCE_TYPE , LEGAL_ENTITY_NAME , INVOICING_ENTITY , LINKED_ACCOUNT , OPERATION , PLATFORM , PURCHASE_TYPE , SERVICE , TENANCY , RECORD_TYPE , and USAGE_TYPE .

Comment (iandees) [reply README.md:73]: Nice, thanks!

**Pull request merged**